monai/networks/utils.py

```diff
@@ -1164,3 +1166,60 @@ def freeze_layers(model: nn.Module, freeze_vars=None, exclude_vars=None):
                 warnings.warn(f"The exclude_vars includes {param}, but requires_grad is False, change it to True.")
 
     logger.info(f"{len(frozen_keys)} of {len(src_dict)} variables frozen.")
+
+
+def get_pretrained_resnet_medicalnet(
```

Comment:
If this function is only used for resnet, I would suggest moving it to `nets.resnet`.